recipes/rapidjson/all/patches/0001-gcc14.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/include/rapidjson/document.h b/include/rapidjson/document.h
+index e3e20dfb..b0f1f70b 100644
+--- a/include/rapidjson/document.h
++++ b/include/rapidjson/document.h
+@@ -316,8 +316,6 @@ struct GenericStringRef {
+ 
+     GenericStringRef(const GenericStringRef& rhs) : s(rhs.s), length(rhs.length) {}
+ 
+-    GenericStringRef& operator=(const GenericStringRef& rhs) { s = rhs.s; length = rhs.length; }
+-
+     //! implicit conversion to plain CharType pointer
```

Comment:
Looking at the commit history, it seems there was another change immediately after which explicitly disabled the copy assignment operator by making it private, perhaps it would be a good idea to include that change too if we added this? (https://github.com/Tencent/rapidjson/commits/master/?after=ab1842a2dae061284c0a62dca1cc6d5e7e37e346+734, specifically this commit: https://github.com/Tencent/rapidjson/commit/862c39be371278a45a88d4d1d75164be57bb7e2d).